Develop

steveforde opened the pull request:
[DEMO LINK](https://steveforde.github.io/Kickstarter/)

lerastarynets reviewed (changes requested):
Pls watch the video review :)

https://github.com/mate-academy/Kickstarter/assets/77551820/be8b314e-d36d-44f3-bbb9-74c6d23b948d

steveforde commented:
Hi lera
I made changes
But the sliders on mobile devices was working for me and not on text
I hope it works for you
I made other changes
Thanks

> On 11 Sep 2023, at 17:09, Lera ***@***.***> wrote:
> 
> 
> @lerastarynets requested changes on this pull request.
> 
> Pls watch the video review :)
> 
> https://github.com/mate-academy/Kickstarter/assets/77551820/be8b314e-d36d-44f3-bbb9-74c6d23b948d
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/mate-academy/Kickstarter/pull/332#pullrequestreview-1620359788>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6EFVOPPZILFTIMTQCSNDZLXZ4ZRZANCNFSM6AAAAAA4TPGPRI>.
> You are receiving this because you authored the thread.
>

lerastarynets reviewed (changes requested):
Well done in fixing majority of issues, let's try to eliminate the rest of issues)

https://github.com/mate-academy/Kickstarter/assets/77551820/74cc08e8-0a43-4b67-8279-a9b30822bfb4

steveforde commented:
Hi lera
I fixed the email form
The sliders on mobile devices works for me
I can’t replicate your issue I’m not sure how to resolve it because it works for me

￼

> On 12 Sep 2023, at 08:43, Lera ***@***.***> wrote:
> 
> 
> @lerastarynets requested changes on this pull request.
> 
> Well done in fixing majority of issues, let's try to eliminate the rest of issues)
> 
> https://github.com/mate-academy/Kickstarter/assets/77551820/74cc08e8-0a43-4b67-8279-a9b30822bfb4
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/mate-academy/Kickstarter/pull/332#pullrequestreview-1621536814>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6EFVOIYQJKATMKET56DIDDX2AHBDANCNFSM6AAAAAA4TPGPRI>.
> You are receiving this because you authored the thread.
>

steveforde commented:
I moved sliders down
Im not sure if it be acceptable to you Lera
I said id try

> On 12 Sep 2023, at 09:28, Stephen Forde ***@***.***> wrote:
> 
> Hi lera
> I fixed the email form
> The sliders on mobile devices works for me
> I can’t replicate your issue I’m not sure how to resolve it because it works for me
> 
> <Screenshot 2023-09-12 at 09.24.38.png>
> 
>> On 12 Sep 2023, at 08:43, Lera ***@***.***> wrote:
>> 
>> 
>> @lerastarynets requested changes on this pull request.
>> 
>> Well done in fixing majority of issues, let's try to eliminate the rest of issues)
>> 
>> https://github.com/mate-academy/Kickstarter/assets/77551820/74cc08e8-0a43-4b67-8279-a9b30822bfb4
>> 
>> —
>> Reply to this email directly, view it on GitHub <https://github.com/mate-academy/Kickstarter/pull/332#pullrequestreview-1621536814>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A6EFVOIYQJKATMKET56DIDDX2AHBDANCNFSM6AAAAAA4TPGPRI>.
>> You are receiving this because you authored the thread.
>> 
>

lerastarynets reviewed (changes requested):
Unfortunately issue with the slider on mobile remains(( pls fix

steveforde commented:
God help meSent from my iPhoneOn 13 Sep 2023, at 11:52, Lera ***@***.***> wrote:﻿
@lerastarynets requested changes on this pull request.

Unfortunately issue with the slider on mobile remains(( pls fix

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

anastasiia-tilikina reviewed (approved):
Wery well done!

steveforde commented:
Thanks AnastasliaSent from my iPhoneOn 18 Sep 2023, at 17:10, Anastasiia Tilikina ***@***.***> wrote:﻿
@anastasiia-tilikina approved this pull request.

Wery well done!

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
